superset/assets/javascripts/explore/components/ExploreViewContainer.jsx

```diff
@@ -53,6 +53,7 @@ class ExploreViewContainer extends React.Component {
     }
     // if any control value changed and it's an instant control
     if (Object.keys(np.controls).some(key => (np.controls[key].renderTrigger &&
+      typeof this.props.controls[key] !== 'undefined' &&
```

Comment:
undefined is primitive value, so we can use
`this.props.controls[key]  !== undefined &&`